pkg/odo/cli/pipelines/wizard.go

```diff
@@ -90,6 +114,46 @@ func (io *WizardParameters) Complete(name string, cmd *cobra.Command, args []str
 	return nil
 }
 
+func checkBootstrapDependencies(io *WizardParameters, kubeClient kubernetes.Interface) error {
+
+	client := utility.NewClient(kubeClient)
+	log.Progressf("\nChecking dependencies\n")
+
+	sealedSpinner := log.Spinner("Checking if Sealed Secrets is installed at kube-system namespace")
+	err := client.CheckIfSealedSecretsExists(sealedSecretsNS+"s", sealedSecretsName)
```

Comment:
This (and the other case) should be able to determine if the k8s objects are "not found" vs "unable to contact the cluster" and report better messages.

Reply:
Updated to throw different errors for "not found" vs "other cluster errors"